TramsDataApi/Migrations/TramsDb/20221208133423_sdd-newcols-datamigrationmk2.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace TramsDataApi.Migrations.TramsDb
+{
+    public partial class sddnewcolsdatamigrationmk2 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
```

Comment:
For my information why are you adding these columns?

Reply:
because we need them for data conversion

Reply:
So will the local authority now be entered by the school?